### DISCUSSION TITLE
Unfortunately it opens at startup

### DISCUSSION THREAD

**Discussion by Feedback1000** [chhoumann/quickadd#638] [Q&A]
Hello.
I created a template. Strangely, it runs every time Obsidian restarts.
Can someone tell me how I can get around this?
Thank you for your help.

**Comment by chhoumann** [comment]
Hi @Feedback1000,

Did you by chance enable this feature in the Macros manager? 
![image](https://github.com/chhoumann/quickadd/assets/29108628/77ac6f8d-5344-4dd5-9aa1-fb1ba08aba38)

Then that'd be the cause.
